app/assets/javascripts/index_map.js

```diff
@@ -12,12 +12,19 @@ function initMap() {
 
   var unit_markers = [];
   var infowindows = [];
-  var unit = {};
+  var unit = [];
   var window_content;
+  var heatMapPoint, heatMapData = [];
 
   for (var i in gon.rental_units) {
-    unit = {lat: gon.rental_units[i].latitude,
-                     lng: gon.rental_units[i].longitude};
+    unit[i] = new google.maps.LatLng(gon.rental_units[i].latitude,
+                                     gon.rental_units[i].longitude);
+
+    // unit[i] = {lat: gon.rental_units[i].latitude,
+    //                  lng: gon.rental_units[i].longitude};
+
+    heatMapPoint = {location: unit[i], weight: gon.rental_units[i].monthly_rent};
```

Comment:
Line is too long.<br>Identifier 'rental_units' is not in camel case.<br>Identifier 'monthly_rent' is not in camel case.<br>'gon' is not defined.